packages/bpk-component-image/src/BpkBackgroundImage.js

```diff
@@ -55,15 +52,7 @@ class BpkBackgroundImage extends Component<BpkBackgroundImageProps> {
   onBackgroundImageLoad: () => mixed;
 
   static propTypes = {
-    aspectRatio: widthHeightAspectRatioPropType,
```

Comment:
Now that we have removed `width` and `height` we no longer need to have a custom prop type for `aspectRatio` and can set it to be a number and set it to be required properly now.